WordPress/Classes/ViewRelated/Gutenberg/Layout Picker/FilterableCategoriesViewController.swift

```diff
@@ -64,6 +69,9 @@ class FilterableCategoriesViewController: CollapsableHeaderViewController {
         tableView.separatorStyle = .singleLine
         tableView.separatorInset = .zero
         tableView.showsVerticalScrollIndicator = false
+        tableView.rowHeight = UITableView.automaticDimension
+        tableView.estimatedRowHeight = UITableView.automaticDimension
```

Comment:
I think these two are not necessary, as in both cases the default value is `UITableView.automaticDimension` ([1](https://developer.apple.com/documentation/uikit/uitableview/1614852-rowheight), [2](https://developer.apple.com/documentation/uikit/uitableview/1614925-estimatedrowheight))

Reply:
Good call @Gio2018 - thanks for catching that. I'll get these wiped. 👍

Reply:
This is resolved with commit 851270f66d95f25976b5656839e007402d2d4e16.